tq/transfer_queue.go

```diff
@@ -633,7 +634,12 @@ func (q *TransferQueue) run() {
 
 // canRetry returns whether or not the given error "err" is retriable.
 func (q *TransferQueue) canRetry(err error) bool {
-	return errors.IsRetriableError(err)
+	switch cause := errors.Cause(err).(type) {
```

Comment:
That's a good point. Though, I don't think the `errors` package strictly has to be about type inference. The `Temporary()` and `Timeout()` methods didn't exist in Go, otherwise we would've used them. I'd prefer to move this to `errors.IsRetriableError()`, but don't have strong feelings. @ttaylorr any thoughts?